cdk/lib/recipes-backend.ts

```diff
@@ -229,6 +233,21 @@ export class RecipesBackend extends GuStack {
 			],
 		});
 
+		new Rule(this, 'ReindexConnection', {
+			eventBus,
+			description: `Connect recipe responder ${this.stage} to recipes-reindex`,
+			eventPattern: {
+				source: ['recipes-reindex'],
```

Comment:
Should we be switching on event detail, instead?

Reply:
I don't think it's necessary in this case - the criterion is that the event is from the recipes reindex process without any other details needed so switching on `source` seems appropriate to me